akka-cluster/src/test/scala/akka/cluster/HeartbeatNodeRingSpec.scala

```diff
@@ -35,15 +35,25 @@ class HeartbeatNodeRingSpec extends WordSpec with Matchers {
       }
     }
 
+    "pick specified number of nodes + unreachable as receivers" in {
+      val ring = HeartbeatNodeRing(cc, nodes, Set(dd, aa), 3)
+      ring.myReceivers should be(ring.receivers(cc))
+
+      ring.receivers(aa) should be(Set(bb, cc, dd, ee))
+      ring.receivers(bb) should be(Set(cc, dd, ee, aa))
+      ring.receivers(cc) should be(Set(dd, ee, aa, bb))
+      ring.receivers(dd) should be(Set(ee, aa, bb, cc))
```

Comment:
BTW: I recently prefer mixing in ConversionCheckedTripleEquals and using `a should ===(b)`, because that fails to compile if b is not of the right type. I guess we could just mechanically convert the whole codebase like that.